package.json

```diff
@@ -35,6 +41,12 @@
     "jest": "26.6.3",
     "lerna": "4.0.0",
     "prettier": "2.2.1",
-    "typescript": "4.2.4"
+    "typescript": "4.2.4",
+    "rollup": "1.31.0",
```

Comment:
Any reasons not to use Rollup v2?

Reply:
I forgot to update it after my tests but it behave the same way, updated in https://github.com/algolia/react-recommendations/pull/6/commits/956efe16d5ec7ded275cbac4540230c44dd0904c